pkg/feeds/epochs/epoch.go

```diff
@@ -0,0 +1,84 @@
+package epochs
+
+import (
+	"encoding/binary"
+
+	"github.com/ethersphere/bee/pkg/crypto"
+)
+
+const (
+	maxLevel = 32
+)
+
+// epoch is referencing a slot in the epoch grid
+type epoch struct {
+	start uint64
+	level uint8
+}
+
+// MarshalBinary implements the BinaryMarshaler interface
+func (e *epoch) MarshalBinary() ([]byte, error) {
+	epochBytes := make([]byte, 8)
+	binary.BigEndian.PutUint64(epochBytes, e.start)
+	return crypto.LegacyKeccak256(append(epochBytes, e.level))
+}
+
+// lca calculates the lowest common ancestor epoch given two unix times
```

Comment:
what does `lowest` mean in this context?

Reply:
lowest level epoch that subsumes both times